dashboards/azure-anomaly-detector/azure-anomaly-detector.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "Azure Anomaly Detector",
+  "description": null,
```

Comment:
Could we add a description for the dashboard?

Reply:
for azure services, none of the dashboard has description so far, following the same standard